Mathlib/Combinatorics/SimpleGraph/Subgraph.lean

```diff
@@ -644,6 +644,9 @@ lemma map_monotone : Monotone (Subgraph.map f) := fun _ _ ↦ map_mono
 theorem map_sup (f : G →g G') (H₁ H₂ : G.Subgraph) : (H₁ ⊔ H₂).map f = H₁.map f ⊔ H₂.map f := by
   ext <;> simp [Set.image_union, map_adj, sup_adj, Relation.Map, or_and_right, exists_or]
 
+@[simp] lemma map_iso_top {H : SimpleGraph W} (e : G ≃g H) : Subgraph.map e.toHom ⊤ = ⊤ := by
```

Comment:
I think this generalises to surjective embeddings? at least syntactically that seems more general

Reply:
A surjective embedding is the same thing as an isomorphism, no? I don't think we want to "generalise" every single lemma from isomorphisms to surjective embeddings

Reply:
Oops, I meant surjective to the support of H, but anyway that's minor